api/src/pcapi/core/mails/transactional/pro/reminder_before_event_to_pro.py

```diff
@@ -0,0 +1,27 @@
+from babel.dates import format_date
+
+from pcapi.core import mails
+from pcapi.core.mails.models.sendinblue_models import SendinblueTransactionalEmailData
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.core.offers.models import Stock
+from pcapi.utils.date import get_time_formatted_for_email
+from pcapi.utils.mailing import get_event_datetime
+
+
+def get_reminder_7_days_before_event_email_data(stock: Stock) -> SendinblueTransactionalEmailData:
+    return SendinblueTransactionalEmailData(
+        template=TransactionalEmail.REMINDER_7_DAYS_BEFORE_EVENT_TO_PRO.value,
+        params={
+            "OFFER_NAME": stock.offer.name,
+            "VENUE_NAME": stock.offer.venue.publicName or stock.offer.venue.name,
+            "EVENT_DATE": format_date(get_event_datetime(stock), format="full", locale="fr"),
+            "EVENT_HOUR": get_time_formatted_for_email(stock.beginningDatetime),
+            "BOOKING_COUNT": stock.dnBookedQuantity,
+        },
+    )
+
+
+def send_reminder_7_days_before_event_to_pro(stock: Stock) -> bool:
+    recipient = stock.offer.bookingEmail or stock.offer.venue.bookingEmail
```

Comment:
Ces 2 champs `bookingEmail` sont nullables, on doit traiter ce cas : 

```suggestion
    recipient = stock.offer.bookingEmail or stock.offer.venue.bookingEmail
    if not recipient:
        return True
```

Reply:
Done